File: src/services/types/models/UserResponse.ts
// UserResponse.ts
// ✅ Type représentant les données reçues du backend pour un utilisateur

export interface UserResponse {
  id: number; // Identifiant unique
  role: string; // Rôle de l'utilisateur (ex: ADMIN, USER)
  firstName: string; // Prénom
  lastName: string; // Nom de famille
  email: string; // Adresse email
  active: boolean; // Statut actif/inactif
}

File: src/main/java/com/gaye/dotoApp/dto/UserResponse.java
package com.gaye.dotoApp.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@AllArgsConstructor
@NoArgsConstructor
@Builder
@Data
public class UserResponse {

    private Integer id;
    private String role;
    private String firstName;
    private String lastName;
    private String email;
    private boolean active;
}

